### PR TITLE
 Made linq to xml and linq to sql queries

### PR DESCRIPTION
**Pull request by INIage**
Issue #167 

- type login and password in app.config
- stylecop without headers

### CODE REVIEW

File: HW5/Task/Task/LinqSamples.cs [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+        {
+            var orderDate = this.dataSource.Customers.SelectMany(c => c.Orders.Select(o => o.OrderDate));
+
+            var query1 = orderDate
+                .GroupBy(
+                    od => od.Month, 
+                    (key, item) => new
+                    {
+                        Month = key,
+                        Orders = item.Count()
```
Comment (KorolS) [inline HW5/Task/Task/LinqSamples.cs]: We need there annual  average

File: HW5/Task/Task/LinqSamples.cs [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+                        AverageProfit = c.Orders.Length == 0 ? 0 : c.Orders.Average(o => o.Total),
+                        AverageIntensity = c.Orders.Length == 0 ? 0 : c.Orders.Count()
+                    })
+                .GroupBy(
+                    c => c.City,
+                    (key, item) => new
+                    {
+                        City = key,
+                        AverageProfit = Math.Round(item.Average(i => i.AverageProfit)),
+                        AverageIntensity = item.Average(i => i.AverageIntensity)
```
Comment (KorolS) [inline HW5/Task/Task/LinqSamples.cs]: I don't think that this point is correct

Comment (KorolS) [reply HW5/Task/Task/LinqSamples.cs]: For ex:
customer1
order 1=100
order 2=200
avg =(100+200)/2=150

customer2
order1= 100
order2=200
order3=300
avg=(100+200+300)/3=200

avg by all orders must be=(100+200+100+200+300)/5=180
or by average =(150*2+200*3)/5=180
and in your variant you get avg= (150+200)/2=175 - that's not right

File: HW5/Task/Task/LinqSamples.cs [hunk 3]
```diff
@@ -0,0 +1,389 @@
...
+        [Title("Task 4")]
+        [Description("This sample return customer's name with first order's date")]
+        public void Linq6()
+        {
+            var query = this.dataSource.Customers
+                .Where(c => c.Orders.Length > 0)
+                .Select(c => new
+                {
+                    CompanyName = c.CompanyName,
+                    FirstOrder = c.Orders.Min(o => o.OrderDate)
```
[the remark this answers is not in the document]
Comment (INIage) [reply HW5/Task/Task/LinqSamples.cs]: I print all date, by if you want i split it

File: HW5/Task/Task/LinqSamples.cs (hunk 1 above)
Comment (INIage) [reply HW5/Task/Task/LinqSamples.cs]: do you mean i have to divide this count by years quantity?

Comment (KorolS) [reply HW5/Task/Task/LinqSamples.cs]: Exactly

**Pull request merged**